client/store/store.js

```diff
@@ -48,11 +49,56 @@ const rootReducer = combineReducers({
   userStore,
 });
 
-// configure persist store and whitelist reducers
+function getUserEncryptionKey() {
+  return process.env.ENCRYPTION_KEY ?? "fd2aafcbf5c604e2613b68e9ef794dc5a18a2715927e9a968d3054c1f2be345a"
```

Comment:
should we just generate instead of hardcoding?

Reply:
the problem in generating is you can generate the key while encrypting because the state of the user is plain text 
but when you want to decrypt you need user information but the user information itself is encrypted then how you'll get the key?